### PR TITLE
Release protocoll

### PR DESCRIPTION
**Pull request by asztalosors**
The automated release protocol was created and some bugs that appeared were fixed. Branch is ready to be merged.

### CODE REVIEW

File: RENATE-OD.info [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        <code_version>1.2.0</code_version>
+        <code_install_requirement>requirements.txt</code_install_requirement>
+        <code_package>renate</code_package>
+        <code_git_link>https://github.com/gergopokol/renate-od.git</code_git_link>
+        <classifier_audience>Intended Audience :: Science/Research</classifier_audience>
+        <classifier_natural_language>Natural Language :: English</classifier_natural_language>
+        <classifier_programming_language>Programming Language :: Python :: 3</classifier_programming_language>
+        <classifier_topic>"Topic :: Scientific/Engineering :: Physics"</classifier_topic>
+    </body>
+</xml>
```
Comment (gergopokol) [inline RENATE-OD.info:17]: EOL to be added.

File: utility/manage.py [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+            raise TypeError('The expected variable type for Version control is str.')
+        version = value.split(sep='.')
+        if not version.__len__() == 3:
+            raise ValueError('The expected data value is point delimited with 3 expressions. Ex: 1.1.0 ')
+        for val in version:
+            assert isinstance(int(val), int), 'Delimited values are expected to be convertible to <int>.'
+        self.major_version = int(version[0])
+        self.minor_version = int(version[1])
+        self.bugfix_version = int(version[2])
+        self.version = version
```
Comment (gergopokol) [inline utility/manage.py:25]: version list is redundant. to be removed

File: utility/manage.py [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+    def _relocate_benchmarks_from_actual_to_archive_on_server(self):
+        for test_case in self.test_cases:
+            file_location = self.test_path + '/' + self.actual_folder + '/' + test_case
+            file_placement = self.test_path + '/' + self.archive_folder + '/' + \
+                             str(self.info.code_version) + '/' + test_case
+            self._update_xml_beamlet_source(file_location+self.data_type['xml'], file_placement+self.data_type['h5'])
+            for server in self.server_type:
+                for extension in self.data_type.keys():
+                    self._move_to_server(server=server, local_path=file_location+self.data_type[extension],
```
Comment (gergopokol) [inline utility/manage.py:199]: to be renamed to _upload_to_server()

File: utility/manage.py [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+        self._upload_actual_benchmarks()
+        self._clean_up()
+
+    @staticmethod
+    def _update_xml_beamlet_source(xml_file, h5_path):
+        param = GetData(data_path_name=xml_file).data
+        param.getroot().find('body').find('beamlet_source').text = h5_path
+        param.write('data/' + xml_file)
+
+    def _move_to_server(self, server, local_path, server_path):
```
Comment (gergopokol) [inline utility/manage.py:234]: to be renamed as above.

**Review by gergopokol — CHANGES_REQUESTED** [review]
Minor changes requested.

File: RENATE-OD.info (hunk 1 above)
Comment (asztalosors) [reply RENATE-OD.info:17]: Done. In: 93a074b

File: utility/manage.py (hunk 3 above)
Comment (asztalosors) [reply utility/manage.py:199]: Done. In: ab00023

File: utility/manage.py (hunk 4 above)
Comment (asztalosors) [reply utility/manage.py:234]: Done. In: ab00023

File: utility/manage.py (hunk 2 above)
Comment (asztalosors) [reply utility/manage.py:25]: Done. In: a3f1cdb

Comment (gergopokol) [reply utility/manage.py]: Way prettier!

Comment (asztalosors) [reply utility/manage.py]: Well. Thank you!

**Review by gergopokol — APPROVED** [review]

**Pull request merged**